src/pages/podcasts.js

```diff
@@ -1,11 +1,26 @@
+import PageLayout from '../components/PageLayout';
 import React from 'react';
-import cx from 'classnames';
 import SEO from '../components/Seo';
-import PageLayout from '../components/PageLayout';
+import cx from 'classnames';
 import podcastsHeader from '../images/podcasts/podcasts.jpg';
 import styles from './podcasts.module.scss';
 
 const PodcastsPage = () => {
+  const podcastIds = ['1225223', '1677727', '1677670'];
+
+  const podcastEmbeds = podcastIds.map((podcastId) => {
+    return (
+      <section className={cx(styles.section, styles.player)} key={podcastId}>
+        <div>
+          <iframe
+            title="buzzsprout"
```

Comment:
Not sure if this matters or not, but now that we have 3 podcast sections, these all have the same `title` attribute. Should this reflect what the podcast is?